utils/training.py

```diff
@@ -141,24 +138,26 @@ def train(model: ContinualModel, dataset: ContinualDataset, args: Namespace) ->
                 if args.debug_mode and i > 3:
                     break
                 if hasattr(dataset.train_loader.dataset, 'logits'):
```

Comment:
Not releted directly to this commit: there is a lot of if else statements in this code right now. In the future we should think about doing something about this. I don't know what this logits attribute of datasets is used for. As far as know Dark Exprience Replay uses the logits that are stored in the buffer. Do you know what this logits in the dataset are for?

Reply:
The logits are used as an additional term in the loss function. According to my understanding, the logits are used as soft targets instead of the actual hard labels. The difference between the logits from the buffer and the current model output is used as the additional loss term.